### PR TITLE
feat: [PIE-2639]: evaluate policy icon

### PR DESCRIPTION
**Pull request by AshwinBhatkal-Harness**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Review by chiragp1212 — APPROVED** [review]

File: src/icons/evaluate-policy.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8 5C7.44772 5 7 5.44772 7 6V21.8175C7 22.1821 7.19853 22.5179 7.51808 22.6937L15.5097 27.0891C15.8121 27.2554 16.1788 27.254 16.4799 27.0855L24.3167 22.7C24.6326 22.5232 24.8283 22.1894 24.8283 21.8273V14.133C24.8283 13.5807 25.2761 13.133 25.8283 13.133C26.3806 13.133 26.8283 13.5807 26.8283 14.133V21.8273C26.8283 22.9135 26.2412 23.9149 25.2934 24.4453L17.4566 28.8308C16.5532 29.3364 15.453 29.3404 14.5458 28.8415L6.55424 24.4461C5.5956 23.9188 5 22.9115 5 21.8175V6C5 4.34315 6.34315 3 8 3H18.0564C18.6087 3 19.0564 3.44772 19.0564 4C19.0564 4.55228 18.6087 5 18.0564 5H8Z" fill="#4F5162"/>
```
Comment (vkbansal) [inline src/icons/evaluate-policy.svg:2]: change fill to `currentColor`

File: src/icons/evaluate-policy.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M26.5411 6.52923C26.9316 6.91976 26.9316 7.55292 26.541 7.94344L16.1477 18.3366L10.5864 12.7754C10.1958 12.3849 10.1958 11.7517 10.5863 11.3612C10.9769 10.9707 11.61 10.9707 12.0006 11.3612L16.1477 15.5082L25.1268 6.52921C25.5174 6.13869 26.1505 6.1387 26.5411 6.52923Z" fill="#4F5162"/>
```
Comment (vkbansal) [inline src/icons/evaluate-policy.svg:3]: change fill to `currentColor`

**Review by vkbansal — CHANGES_REQUESTED** [review]

File: src/icons/evaluate-policy.svg [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (AshwinBhatkal-Harness) [reply src/icons/evaluate-policy.svg]: Removed

File: src/icons/evaluate-policy.svg (hunk 2 above)
Comment (AshwinBhatkal-Harness) [reply src/icons/evaluate-policy.svg]: Changed

File: src/icons/evaluate-policy.svg (hunk 1 above)
Comment (AshwinBhatkal-Harness) [reply src/icons/evaluate-policy.svg]: Changed

**Review by vkbansal — APPROVED** [review]

**Pull request merged**